.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,35 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: 'The name of the tarantool build artifact'
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: 'Clone the vshard module'
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/vshard
+          # Fetch the entire history for all branches and tags. It is needed for
+          # upgrade testing.
+          fetch-depth: 0
+          # Enable recursive submodules checkout as test-run git module is used
+          # for running tests.
+          submodules: recursive
+      - name: 'Download the tarantool build artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+      - name: 'Install tarantool'
+        run: sudo dpkg -i tarantool*.deb
```

Comment:
We can skip installing of a source package (just to don't so unnecessary actions):

https://github.com/tarantool/smtp/blob/4831a653fe9f8c66b23e44876d7805f93f2eb492/.github/workflows/packaging.yml#L157-L165

Reply:
What also doubts me: whether Ubuntu Focal on a GitHub hosted runner always contains all necessary runtime dependencies of a tarantool package.

Reply:
Yaroslav shows me logs:

```
Run sudo dpkg -i tarantool*.deb
  sudo dpkg -i tarantool*.deb
  shell: /usr/bin/bash -e {0}
Selecting previously unselected package tarantool-common.
(Reading database ... 231562 files and directories currently installed.)
Preparing to unpack tarantool-common_2.10.0~beta1.138.dev-1_all.deb ...
Unpacking tarantool-common (2.10.0~beta1.138.dev-1) ...
Selecting previously unselected package tarantool-dev.
Preparing to unpack tarantool-dev_2.10.0~beta1.138.dev-1_amd64.deb ...
Unpacking tarantool-dev (2.10.0~beta1.138.dev-1) ...
Selecting previously unselected package tarantool.
Preparing to unpack tarantool_2.10.0~beta1.138.dev-1_amd64.deb ...
Unpacking tarantool (2.10.0~beta1.138.dev-1) ...
Setting up tarantool-common (2.10.0~beta1.138.dev-1) ...
Created symlink /etc/systemd/system/multi-user.target.wants/tarantool.service → /lib/systemd/system/tarantool.service.
Setting up tarantool (2.10.0~beta1.138.dev-1) ...
update-alternatives: using /usr/bin/tarantool to provide /usr/bin/lua (lua-interpreter) in auto mode
Setting up tarantool-dev (2.10.0~beta1.138.dev-1) ...
Processing triggers for systemd (245.4-4ubuntu3.13) ...
Processing triggers for man-db (2.9.1-1) ...
```

It seems, it actually works. However I would leave a TODO comment like [this one](https://github.com/tarantool/smtp/blob/4831a653fe9f8c66b23e44876d7805f93f2eb492/.github/workflows/packaging.yml#L159).